src/modules/auth/services.js

```diff
@@ -0,0 +1,18 @@
+export default {
+
+  authenticate(socialId, scope) {
```

Comment:
You should never export objects like that because eslint can validate its existence.
This should be written as `export const authenticate = (socialId, scope) [etc]`